echo-web/src/main/java/com/netflix/spinnaker/echo/scheduler/TriggerDescription.java

```diff
@@ -0,0 +1,25 @@
+package com.netflix.spinnaker.echo.scheduler;
+
+import lombok.Data;
+import org.hibernate.validator.constraints.NotBlank;
```

Comment:
How is this annotation used, given that we're not using Hibernate?

Reply:
it was used in `ScheduledActionController` (with `@Validated`) but I just removed it and doing manual validation now because it was giving me a headache with UTs https://github.com/spinnaker/echo/pull/481/files#diff-2bcb5815f22def5c47e7f2b2e78ed6ffR87

Reply:
Hibernate is just an implementation of the JSR-303 spec. We don't use hibernate, but their validation library is pretty ubiquitous (and also doesn't pull in hibernate proper as a dependency).